deblur/workflow.py

```diff
@@ -460,22 +547,33 @@ def launch_workflow(seqs_fp, working_dir, read_error, mean_error, error_dist,
                            "%s.derep" % basename(output_trim_fp))
     dereplicate_seqs(seqs_fp=output_trim_fp,
                      output_fp=output_derep_fp,
-                     min_size=min_size,
-                     uc_output=True)
+                     min_size=min_size)
     # Step 3: Remove artifacts
     output_artif_fp = remove_artifacts_seqs(seqs_fp=output_derep_fp,
                                             ref_fp=ref_fp,
                                             working_dir=working_dir,
                                             ref_db_fp=ref_db_fp,
                                             negate=negate,
                                             threads=threads)
+    if not output_artif_fp:
```

Comment:
we don't want to abort the whole pipeline if one sample fails (can happen if sample had 1 read, etc.).
Added a warning to the user instead